configuration/wireless/access-point-bridged.md

```diff
@@ -120,6 +120,21 @@ interface br0
 ```
 With this line, interface `br0` will be configured in accordance with the defaults via DHCP. Save the file to complete the IP configuration of the machine.
 
+<a name="wifi-cc-rfkill"></a>
+## Ensure wireless operation
+
+Countries around the world regulate the use of telecommunication radio frequency bands to ensure interference-free operation. The Linux OS [helps](https://wireless.wiki.kernel.org/en/developers/regulatory/statement) users comply with these rules by simply configuring applications with a 2-letter "WiFi country code", e.g. `US` for a computer used in the United States.
```

Comment:
I'm not quite sure what you meant here, but this sounds like it's the OS that's doing the configuration, rather than the user :wink: 
Would "The Linux OS helps users comply with these rules by allowing applications to be configured with a 2-letter..." work better? (and I wonder if the link would be better-placed on "comply" rather than "helps"?)

Reply:
Or possibly no link at all. If you clicked it, you see it is a sort of pledge of allegiance of kernel devs to regulatory bodies. It's a bit political and in any case far remote from the concerns of a user who just wants to setup an AP.
Shall I remove the link altogether?
(I will rephrase. Thanks)

Reply:
While I'm at it. I used the "US" country code because the country code for UK is "GB"... I thought "... e.g. `GB` for a computer used in the United Kingdom." didn't provide the best demonstration of how simple compliance is supposed to be.
If using `GB`/United Kingdom is clear enough then I could change that. The actual hostapd.conf later on says `country_code=GB` (and I think that's fine, there.)

Reply:
> Shall I remove the link altogether?

I dunno - it _does_ provide a good explanation of why it's "off by default" and has to be manually unblocked.

Still something I know very little about though, so I'll leave the decision to @XECDesign

Reply:
Rewritten sentence now. Link still in, I don't dislike it, finally.
Same in both bridged and routed files.